src/components/BeaconResultTileDetailsV2.vue

```diff
@@ -148,35 +294,61 @@ export default {
       else this.display = true;
     },
     checkForPublicDatasets: function(result) {
-      if (
-        result.info &&
-        result.info.accessType &&
-        result.info.accessType == "PUBLIC"
-      )
-        return true;
+
+      // commented to reduce error messages FIX LATER if possible. Beacon2 does not seem to return access type
```

Comment:
if we cannot determine access type for beacon we default to `unknown`, how about that ?
if the keys necessary to determine accesstype exists then we can use the code below